tsrc/test/cli/test_sync.py

```diff
@@ -127,10 +127,10 @@ def test_changing_branch(tsrc_cli, git_server, workspace_path, message_recorder)
     assert message_recorder.find("not on the correct branch")
```

Comment:
You've ported the tests that were using tags, but we now need at least one test using sha1s too